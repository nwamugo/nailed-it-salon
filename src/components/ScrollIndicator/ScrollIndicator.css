.scroll-indicator {
  position: fixed;
  top: 27.4375rem;
  right: 9.9375rem;
  width: 6.6875rem;
  height: 5.1875;
  font-family: 'Roboto', sans-serif;
  font-size: 0.75rem;
  text-transform: uppercase;
  vertical-align: top;
}

.indicator-item {
  margin-bottom: .5rem;
}

.dot {
  position: fixed;
  right: 9.9375rem;
  height: .6625rem;
  width: .6625rem;
  background-color: #f0a07f;
  border-radius: 50%;
  display: inline-block;
}