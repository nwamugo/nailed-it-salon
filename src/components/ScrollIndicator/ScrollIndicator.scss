.scroll-indicator {
  position: fixed;
  top: 43.9rem;
  right: 15.9rem;
  width: 10.7rem;
  height: 8.3rem;
  font-family: 'Roboto', sans-serif;
  font-size: 1.2rem;
  text-transform: uppercase;
  vertical-align: top;

  @media (max-width: 26.75em) {
    display: none;
  }
}

.indicator-item {
  margin-bottom: .8rem;
}

.dot {
  position: fixed;
  right: 15.9rem;
  height: 1.06rem;
  width: 1.06rem;
  background-color: #f0a07f;
  border-radius: 50%;
  display: inline-block;
}