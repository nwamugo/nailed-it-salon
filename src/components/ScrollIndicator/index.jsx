import React from 'react';
import './ScrollIndicator.scss';

const ScrollIndicator = () => {
  return (
    <div className="scroll-indicator">
      <div className="indicator-item">
        Home
        <span className="dot" />
      </div>
      <div className="indicator-item">
        Our Services
        <span className="dot" />
      </div>
      <div className="indicator-item">
        Our Story
        <span className="dot" />
      </div>
      <div className="indicator-item">
        Make an appointment
        <span className="dot" />
      </div>
      <div className="indicator-item">
        Contact
        <span className="dot" />
      </div>
    </div>
  );
};

export default ScrollIndicator;
