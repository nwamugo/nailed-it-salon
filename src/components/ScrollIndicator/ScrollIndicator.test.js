import React from 'react';
import { shallow } from 'enzyme';
import ScrollIndicator from './index';

describe('ScrollIndicator', () => {
  it('should render correctly', () => {
    const component = shallow(<ScrollIndicator />);

    expect(component).toMatchSnapshot();
  });
});
