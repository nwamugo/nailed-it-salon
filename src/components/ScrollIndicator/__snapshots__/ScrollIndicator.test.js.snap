// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollIndicator should not have regressed 1`] = `
<div
  className="scroll-indicator"
>
  <div
    className="indicator-item"
  >
    Home
    <span
      className="dot"
    />
  </div>
  <div
    className="indicator-item"
  >
    Our Services
    <span
      className="dot"
    />
  </div>
  <div
    className="indicator-item"
  >
    Our Story
    <span
      className="dot"
    />
  </div>
  <div
    className="indicator-item"
  >
    Make an appointment
    <span
      className="dot"
    />
  </div>
  <div
    className="indicator-item"
  >
    Contact
    <span
      className="dot"
    />
  </div>
</div>
`;
